src/main/java/com/ft/methodearticlemapper/transformation/EomFileProcessor.java

```diff
@@ -60,9 +60,14 @@
         PREVIEW
     }
 
+    interface Type {
+//        String ARTICLE = "http://www.ft.com/ontology/content/Article";
```

Comment:
Sometime in the future, we are going to move the whole type determination into the mappers. So I though I'd be a good sport and add the type constants ahead of time. However, IDEA kept bugging me that the values are not used, so I commented them out :)

I can either remove the comments or remove the fields entirely, since it might take a while until they will be used. Up to you.